lib/twitter/media/animatedgif.rb

```diff
@@ -0,0 +1,9 @@
+require 'twitter/identity'
```

Comment:
any particular reason why you required `twitter/identity` here? It's not used in this class.